Mlem/App/Views/Pages/Instance/InstanceDetailsView.swift

```diff
@@ -0,0 +1,296 @@
+                            Label(
+                                "This probably contains foul language.",
+                                systemImage: Icons.warning
+                            )
+                            .foregroundStyle(.orange)
+                        }
+                    }
+                    .font(.footnote)
+                    .frame(maxWidth: .infinity, alignment: .leading)
+                    .padding(12)
+                    .contentShape(.rect)
+                    .onTapGesture {
+                        withAnimation(.easeInOut(duration: 0.2)) {
+                            showingSlurRegex.toggle()
+                        }
+                    }
+                }
+                if let feedType = instance.defaultFeed_ {
+                    Divider()
+                    settingRow(
+                        "Default Feed Type (Desktop)",
+                        systemImage: Icons.feeds,
+                        value: feedType.rawValue
+                    )
+                }
+            }
+            .frame(maxWidth: .infinity)
+            .background(Color(uiColor: .secondarySystemGroupedBackground))
+            .cornerRadius(AppConstants.largeItemCornerRadius)
+            
+            VStack(alignment: .leading, spacing: 0) {
+                settingRow(
+                    "Show Mod Names in Modlog",
+                    systemImage: Icons.moderation,
+                    value: !(instance.hideModlogNames_ ?? true)
+                )
+                Divider()
+                settingRow(
+                    "Applications Email Admins",
+                    systemImage: Icons.person,
+                    value: instance.emailApplicationsToAdmins_ ?? false
+                )
+                Divider()
+                settingRow(
+                    "Reports Email Admins",
+                    systemImage: Icons.moderationReport,
+                    value: instance.emailReportsToAdmins_ ?? false
+                )
+            }
+            .frame(maxWidth: .infinity)
+            .background(Color(uiColor: .secondarySystemGroupedBackground))
+            .cornerRadius(AppConstants.largeItemCornerRadius)
+        }
+        .padding(.horizontal, 16)
+    }
+    
+    var captchaLabel: String {
+        if let diff = instance.captchaDifficulty_ {
+            return "Yes (\(diff.rawValue.capitalized))"
+        }
+        return "No"
+    }
+    
+    @ViewBuilder func box(spacing: CGFloat = 5, @ViewBuilder content: () -> some View) -> some View {
+        VStack(spacing: spacing) {
+            content()
+        }
+        .padding(.vertical, 10)
+        .frame(maxWidth: .infinity)
+        .background(Color(uiColor: .secondarySystemGroupedBackground))
+        .cornerRadius(AppConstants.largeItemCornerRadius)
+    }
+    
+    @ViewBuilder func settingRow(
+        _ label: String,
+        systemImage: String,
+        value: String,
+        color: Color = .primary
```

Comment:
Or `Palette.main`, not sure how nicely default parameters play with environment

```suggestion
        color: Color = palette.primary
```